test/cases/no_compile_panic_for_arith_wrap.zig

```diff
@@ -0,0 +1,12 @@
+pub const panic = @compileError("");
+
+export fn entry() usize {
+    var x: usize = 0;
+    x +%= 1;
+    x -%= 1;
+    x *%= 1;
+    return x;
+}
+
+// compile
+// output_mode=Obj
```

Comment:
Without including the llvm backend with `// backend=stage2,llvm`, this test will not fail without the fix.

Reply:
Ah, my mistake. I thought LLVM was still the default release backend. I will add `// backend=stage2,llvm` with the test for #18376.